srcpkgs/urlscan/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'urlscan'
+pkgname=urlscan
+version=0.9.0
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="urlscan"
+hostmakedepends="python3-setuptools"
+depends="python3-urwid"
+short_desc="View/select the URLs in an email message or file"
+maintainer="Crestwave <crest.wave@yahoo.com>"
+license="GPL-2"
```

Comment:
use spdx short-identifier

Reply:
Done. I’m confused, though; some packages and the manual do not use that identifier.

Reply:
It is a new "policy" so the manual and most packages still aren't updated.

https://github.com/voidlinux/void-packages/issues/12219